explainers/hidden-content-explainer.md

```diff
@@ -167,11 +167,17 @@ the page, we have added two constraints to beforematch:
    events and guessing which element was scrolled to, the beforematch event
    without this restriction is a stronger and more accurate signal.
 
-2. If the page doesn't reveal the text in response to the beforematch event, the
-   beforematch event will not be fired for the remainder of the lifetime of the
-   document. By "reveal," I mean that the `content-visibility: hidden-matchable`
-   property is removed and there are no other properties applied which would
-   prevent find-in-page from finding the text normally, such as `display: none`.
+2. If the page fails to reveal the active match when the browser tries to scroll
+   to the active match after firing the beforematch event, the beforematch event
+   will be disabled for the remaining lifetime of the document. By "reveal," I
+   mean that the active match fulfills all of these requirements:
+   - The DOM range of the target match is not
+     [collapsed](https://dom.spec.whatwg.org/#range-collapsed) (meaning that the
+     target match was not removed from the DOM.
+   - The `content-visibility` CSS property is not `hidden` or
+     `hidden-matchable`.
+   - The `display` CSS property is not `display: none`.
```

Comment:
The visibility property is inherited, so the text for it is correct. For display none and content-visibility hidden, however, we should say that the element does not have display: none and is not in the subtree of any display: none element

Reply:
done